### ISSUE TITLE
Server crashes with `Cannot read property '__constructor' of undefined`

### ISSUE THREAD

**Issue by chuckjaz** [angular/vscode-ng-language-service#53]
Moved from issue #49:

hi. i just installed the Experimental Release 0.1.0 and everything was working but then it crashed again. 

here is the error message:

```
/Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:47960
        var constructor = type.members['__constructor'];
                                      ^

TypeError: Cannot read property '__constructor' of undefined
    at TypeScriptSymbolQuery.getTemplateRefContextType (/Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:47960:39)
    at TypeScriptSymbolQuery.getTemplateContext (/Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:47935:36)
    at ExpressionDiagnosticsVisitor.visitVariable (/Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:47029:52)
    at VariableAst.visit (/Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:16446:24)
    at visit (/Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:16681:37)
    at /Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:16683:42
    at Array.forEach (native)
    at templateVisitAll (/Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:16682:10)
    at visit (/Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:44866:30)
    at /Users/Kwakes/.vscode/extensions/Google.ng-template-0.1.0/server/node_modules/@angular/language-service/bundles/language-service.umd.js:44827:13
[Error - 2:20:17 PM] Connection to server got closed. Server will not be restarted.

```

**Comment by chuckjaz** [comment]
Tracking in https://github.com/angular/angular/issues/15528

**Comment by chuckjaz** [comment]
Fixed in `0.1.1`
